delta/app/src/main/resources/application-cassandra.conf

```diff
@@ -23,7 +23,7 @@ akka {
       events-by-tag {
         first-time-bucket = "20181213T00:00"
         # This delay helps to order events. Setting this to anything lower than 2s is highly discouraged.
-        eventual-consistency-delay = 10s
+        eventual-consistency-delay = 5s
```

Comment:
This timeout seem low compared to the `tag-write-timeout` and the evaluation timeout, we may miss some events

Reply:
I will introduce it in a subsequent PR.